### PR TITLE
Create Skycoin Next Generation.es.md

### PR DESCRIPTION
**Pull request by Murlocvici**

### CODE REVIEW

File: content/statement/Skycoin Next Generation.es.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+]
++++
+
+Esta es una publicación archivada del hilo bitcointalks del 06 de mayo de 2014.
+
+Cita de: -Greed- el 5 de mayo de 2014, 04:25:46 p.m.+1 para que Skycoin sea verdaderamente
+una moneda de segunda generación. En mi opinión, Bitcoin y todos sus forks son de primera
+generación. Algunas de las cosas sobre las que estamos sentando las bases son de tercera
+generación. Una moneda utilizable, trabajable, con una base de código limpia y una buena
+seguridad, sin minería, sin el 51% de ataque y una razón para usar una moneda que es de
```
Comment (Palvini) [inline content/statement/Skycoin Next Generation.es.md]: sin el 51% de ataque fix

**Comment by Palvini** [comment]
looks good

**Comment by 00glm** [comment]
It's good.

**Comment by gz-c** [comment]
8 sky

**Pull request merged**